third_party/googleapis/google/devtools/build/v1/publish_build_event.proto

```diff
@@ -184,4 +184,8 @@ message PublishBuildToolEventStreamRequest {
   // request. BES only performs this check for events with sequence_number 1
   // i.e. the first event in the stream.
   bool check_preceding_lifecycle_events_present = 7;
+
+  // The number of retry attempts has been made to publish the unique event.
+  // Starts from 0.
+  int32 retry_attempt = 8;
```

Comment:
I think this should be `attempt_number` and start from 1 so that the server can distinguish between "this is the first attempt" and "this is an old version of bazel that doesn't support this field", which are currently both represented as 0.